src/components/elementView/elementTab.tsx

```diff
@@ -610,8 +611,14 @@ class ElementTab extends React.Component<ElementTabProps, ElementViewState> {
               this.setState({ filters: data });
             }}
             shouldUpdateData={this.state.updateTable}
-            disableSorting={this.props.element === ElementType.DATACENTER}
-            disableFiltering={this.props.element === ElementType.DATACENTER}
+            disableSorting={
+              this.props.element === ElementType.DATACENTER ||
+              this.props.element === ElementType.OFFLINE_STORAGE_SITE
+            }
+            disableFiltering={
+              this.props.element === ElementType.DATACENTER ||
+              this.props.element === ElementType.OFFLINE_STORAGE_SITE
+            }
```

Comment:
don't want to show filters/sorting on offline storage sites tab

Reply:
this is leftover from last evolution, do we want to add sorting and filtering functionality? it would be nice to have + be consistent with all our other tables

Reply:
it would be supported from front end, would need to add logic to endpoints though

Reply:
sure I can add that here